src/rpc/blockchain.cpp

```diff
@@ -630,6 +630,23 @@ static CBlock GetBlockChecked(const CBlockIndex *pblockindex)
     return block;
 }
 
```

Comment:
I'd add a comment about what this function is doing (even if it is local to this file and was not present in core PR you are porting)